include/seqan/index/index_fm.h

```diff
@@ -205,6 +205,16 @@ class Index<TText, FMIndex<TSpec, TConfig> >
     typename Fibre<Index, FibreLF>::Type            lf;
     typename Fibre<Index, FibreSA>::Type            sa;
 
+    /*!
+     * @fn FMIndex::Index
+     * @brief Constructor
+     *
+     * @signature Index::Index();
+     * @signature Index::Index(text);
+     *
+     * @param[in] text The text to be indexed.
+     */
```

Comment:
@cpockrandt will add a tag ("suffix") to the FMindex interface s.t. the user will be able to use the FMindex as a suffix-trie and the text does not need to be reversed beforehand. Since the documentation of this should be done in another pull request, the current documentation here is sufficient I think?